apps/automation_crews/dev_crew/agents/frontend_developer/frontend_developer.py

```diff
@@ -1,14 +1,17 @@
 import os
 from textwrap import dedent
 from crewai import Agent
-from tools.file_write import FileWrite
-from tools.file_read import FileRead
+from dev_crew.tools.file_write import FileWrite
+from dev_crew.tools.file_read import FileRead
 from langchain_groq import ChatGroq
 from dotenv import load_dotenv
+
 load_dotenv()
 
 api_key = os.getenv("GROQ_API_KEY")
-llm = ChatGroq(model = "llama-3.1-70b-versatile", api_key = api_key)
+llm = ChatGroq(model="llama-3.1-70b-versatile", api_key=api_key)
```

Comment:
same

Reply:
initialize llm in new created module llm.py and import in rest of the modules.